tests/test_requirements.txt

```diff
@@ -0,0 +1,3 @@
+libsonata
+pandas
+pyarrow
```

Comment:
is numpy missing?

Reply:
IIRC, both libsonata and pandas should require it.